src/recommender.py

```diff
@@ -366,9 +366,11 @@ class RecommendationTask:
 
     _analysis_name = 'recommendation_v2'
     description = 'Get Recommendation'
+    kronos_ecosystems = ['maven']
+    chester_ecosystems = ['npm']
 
     @staticmethod
-    def call_pgm(payload):
+    def call_insights_recommender(payload):
         """Call the PGM model.
```

Comment:
Guess we can update this doc string as well to be consistent with the replacement of `pgm` with `insights_recommender`